.env.example

```diff
@@ -1 +1,4 @@
 DATABASE_URL=postgres://postgres:default_password@localhost:5432/checkpoint
+ADMIN_ADDRESS='0x8C28Cf33d9Fd3D0293f963b1cd27e3FF422B425c'
+TURBO_MONTHLY_PRICE=600
+TURBO_YEARLY_PRICE=6000
```

Comment:
If at some point we need to change it we wont be able to do that by just changing the env vars, that would mess the duration of plans that was paid before price change. I believe we should hardcode these values instead, and later keep the history of pricing also in the code

Reply:
do you want to go back to how it was before? i e hardcoded

Reply:
Now that we have unlimited supply of tests tokens, my request is now obsolete